charts/sn-platform/templates/_helpers.tpl

```diff
@@ -105,4 +105,35 @@ Istio gateway selector
 {{- else }}
 istio: ingressgateway
 {{- end }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+{{/*
+JVM Options
+*/}}
+{{- define "pulsar.jvm.options" -}}
+jvmOptions:
+  memoryOptions:
+  {{- if .configData.PULSAR_MEM }}
+  - {{ .configData.PULSAR_MEM | quote }}
+  {{- else }}
+  {{- with .jvm.memoryOptions }}
```

Comment:
```suggestion
  {{- with .jvm.memoryOptions -}}
```

Reply:
To delete the blank line after it.
```yaml
    jvmOptions:
      memoryOptions:

      - |
        -Xms64m -Xmx128m
      gcOptions:

      - |
        -XX:+UseG1GC
```

Reply:
Thanks for pointing this out, I changed `{{ toYaml` to `{{- toYaml` to align with other statements.